tests/spatial/indexer_perf_test.cc

```diff
@@ -0,0 +1,163 @@
+#include "catch.hpp"
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/box.hpp>
+#include <boost/geometry/geometries/point.hpp>
+#include <boost/geometry/index/rtree.hpp>
+#include <chrono>
+
+#include "s2/base/commandlineflags.h"
```

Comment:
We don't have s2 included, this will fail?

Reply:
This is file is not included in makefile. Its just for performance check between s2 and boost rtree.